transforms/code/license_check/python/README.md

```diff
@@ -0,0 +1,68 @@
+# License Check 
+
+Please see the set of
+[transform project conventions](../../../README.md#transform-project-conventions)
+for details on general project conventions, transform configuration,
+testing and IDE set up.
+
+## Summary 
+
+This filter scans the license column of an input dataset and appends the `license_status` column to the dataset.
+
+The type of the license column can be either string or list of strings. For strings, the license name is checked against the list of approved licenses. For list of strings, each license name in the list is checked against the list of approved licenses, and all must be approved.
+
+If the license is approved, the license_status column contains True; otherwise False. 
+
+## Configuration and command line Options
```

Comment:
Can you discuss these separately as configurations for the Transform and separately the cli parameters that have the lc_ prefix?  You can see how code2parquet does this as a template